brewtils/config.py

```diff
@@ -145,14 +151,18 @@ def load_config(cli_args=None, argument_parser=None, **kwargs):
         sources.append(("kwargs", kwargs))
 
     if cli_args:
-        if not argument_parser:
-            argument_parser = ArgumentParser()
-            spec.add_arguments(argument_parser)
+        if cli_args is True:
+            sources.append("CLI")
```

Comment:
This is a yapconf thing - if you give it a tuple it'll make better error messages. I'll go through and add labels for the 2 sources that don't have them.

Reply:
Just kidding - yapconf requires "special" sources to always be specified as these string values, they can't have a different label. That's OK though, since the labels I was going to put on them were just "command line" and "environment".